### DISCUSSION TITLE
User prompted for password for updating?

### DISCUSSION THREAD

**Discussion by coryl** [sparkle-project/Sparkle#2493] [Q&A]
I currently have these Info.plist settings in v1.0, and just shipped a v1.1 update, so both versions have these values:
```
	<key>SUEnableAutomaticChecks</key>
	<true/>
	<key>SUAllowsAutomaticUpdates</key>
	<true/>
	<key>SUAutomaticallyUpdate</key>
	<true/>
	<key>SUScheduledCheckInterval</key>
	<integer>3600</integer>
```
The behaviour I'm trying to achieve for my app is to simply always download and install updates quietly, without any user interaction.

We shipped our first update to the public. We weren't sure why we weren't able to trigger an immediate download and update, or even within a few hours, but anyway sometime overnight a dialog box appeared indicating an update was ready:
![image](https://github.com/sparkle-project/Sparkle/assets/1355864/769f9920-877e-4af2-bbb3-9a21e01d03f6)

I clicked Install Update and was prompted for a password to update:
![image](https://github.com/sparkle-project/Sparkle/assets/1355864/d36094ea-d1bd-475e-898d-7d85f8c949c6)

So really a couple questions:
- Why didn't the app silently update in the background like in testing?
- Why was I prompted for a password to update? My cofounder did not receive this password prompt for his update.

Thanks

**Comment by zorgiepoo** [comment]
Because that user required authorization to have permission to install and overwrite the update. I explained that [recently here](https://github.com/sparkle-project/Sparkle/issues/2488#issuecomment-1893807452). Of course in this scenario an update cannot be installed silently and Sparkle needs to alert the user of the update more immediately.

**Reply by zorgiepoo** [reply]
Note the update may still be downloaded automatically before presented to the user. In the first screenshot where the update has been downloaded automatically the checkbox to automatically download / install updates should be initially checked, unless you toggled its state afterwards.
